src/core/Table/Table.tsx

```diff
@@ -550,7 +548,10 @@ export const Table = <
           ].getBoundingClientRect().width;
         }
       });
-      dispatch({ type: tableResizeEndAction, columnWidths: newColumnWidths });
+      dispatch({
```

Comment:
minor, but as you gonna have commit, revert this change too... should fit in one line

Reply:
reverted